server/views/pages/recallLastRelease.njk

```diff
@@ -83,9 +83,8 @@
                         ],
                         errorMessage: {
                             text: 'Enter a valid date in the past'
-                        } if formValues.lastReleaseDateTime else undefined
+                        } if errors.lastReleaseDateTime else undefined
```

Comment:
The cypress tests are currently looking for the error block at the top of the page; we can extend them (in a later PR) to look for this error message lower down as well, it would have caught this.